### PR TITLE
Add licence

### PR DESCRIPTION
**Pull request by Reception123**
Not sure if its good

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -319,7 +319,21 @@
 		),
 	),
 
-	// Server
+	     // License
```
Comment (addshore) [inline LocalSettings.php]: indenting for // License looks odd

File: LocalSettings.php [hunk 2]
```diff
@@ -319,7 +319,21 @@
...
+		'default' => '',
+	),
+	'wgRightsText' => array(
+		'default' => 'Creative Commons Attribution Share Alike',
+	),
+	'wgRightsUrl' => array(
+		'default' => 'https://creativecommons.org/licenses/by-sa/3.0/',
+
+	),
+             // Server
```
Comment (addshore) [inline LocalSettings.php]: indenting for // Server looks odd

File: LocalSettings.php [hunk 3]
```diff
@@ -319,7 +319,22 @@
...
+	),
+	'wgRightsText' => array(
+		'default' => 'Creative Commons Attribution Share Alike',
+	),
+	'wgRightsUrl' => array(
+		'default' => 'https://creativecommons.org/licenses/by-sa/3.0/',
+
+	),
+
+        // Server
```
Comment (Southparkfan) [inline LocalSettings.php]: Strange indent

File: LocalSettings.php [hunk 4]
```diff
@@ -319,7 +319,22 @@
...
+	),
+	'wgRightsPage' => array(
+		'default' => '',
+	),
+	'wgRightsText' => array(
+		'default' => 'Creative Commons Attribution Share Alike',
+	),
+	'wgRightsUrl' => array(
+		'default' => 'https://creativecommons.org/licenses/by-sa/3.0/',
+
```
Comment (Southparkfan) [inline LocalSettings.php]: Unwanted newline

File: LocalSettings.php [hunk 5]
```diff
@@ -319,7 +319,22 @@
 		),
 	),
 
-	// Server
+	// License
+	'wgRightsIcon' => array(
+		'default' => "//$wmgUploadHostname/common/skins/common/images/cc-by-sa.png",
```
Comment (Southparkfan) [inline LocalSettings.php]: Set this to: https://meta.miraheze.org/w/resources/assets/licenses/cc-by-sa.png

**Comment by Southparkfan** [comment]
Will merge this in one commit.

**Pull request closed without merging**